app/features/ledger/ConcordiumLedgerClient.ts

```diff
@@ -234,6 +235,20 @@ export default class ConcordiumLedgerClient {
         );
     }
 
+    signAddIdentityProvider(
+        transaction: UpdateInstruction<AddIdentityProvider>,
+        serializedPayload: Buffer,
+        path: number[]
+    ): Promise<Buffer> {
+        return toBuffer(
+            window.ledger.signAddIdentityProvider(
+                stringify(transaction),
```

Comment:
Check if this is necessary as you said it might not be.

Reply:
It was not, I removed it from all the ledger functions.